README.md

```diff
@@ -64,27 +69,35 @@ Describe the rules of scope in JavaScript.
 
 Your Answer:
 ```text
+Scope is a concept referring to what can be accessed from where in JS. If an item that can be named (things like functions or variables) are named without those modifiers and are outside of any function they are considered to be in the "global" scope which by default refers to the "window" in a browser. If functions or variables are called within a function, or if they use the keywords "var" or "function" then they are only accessible within THAT present scope and the ones outside of them. NOT the ones inside them. This is why hoisting works.
 ```
```

Comment:
Apologies if this is what you were getting at, but values are accessible within their current and deeper scopes (e.g., a function within a function).